### ISSUE TITLE
Add version to the shader header?

### ISSUE THREAD

**Issue by Matsilagi** [Chaython/Raytracing-path-tracing-RTGI-Reshade#1]
Could you please add what version of RTGI this shader is?
From what i've looked, it seems to be 0.8.2, just so people atleast knows what version this is.

**Comment by Chaython** [comment]
0.8.0.2
I discussed with the developer and he changed licensing terms on future releases to disallow sharing.
________________________________

**Comment by Chaython** [comment]
Did Marty approve your fork for distribution? Though no licensing terms were provided? If so I will submit the fork here.

**Comment by Matsilagi** [comment]
Yeah he did, lemme just make a fork with that change and then you can merge it.

**Comment by Chaython** [comment]
I was refering to "LateNightBothering" which seems to be a fork and included no licensing terms.
I was going to create a branch with this here.
`/*
	MADE BY MATSILAGI, NOT OFFICIAL, PLEASE DON'T BOTHER MARTY IF IT BACKFIRES, EXPLODES YOUR GPU OR ARTIFACTS IT.
*/

/*
	changelog:

	2.0.0:	added frame count parameter
			added versioning system
			removed common textures - should only be declared if needed
			flipped reversed depth buffer switch by default as most games use this format

*/`
As for adding version number, as per the license used for the files here, I interpret the terms as disallowing modification, however since you state you were given permission I merged your fork.
